main.py

```diff
@@ -0,0 +1,59 @@
+# Создать форму для регистрации пользователей на сайте.
+# Форма должна содержать поля "Имя", "Фамилия", "Email", "Пароль"
+# и кнопку "Зарегистрироваться". При отправке формы
+# данные должны сохраняться в базе данных, а пароль должен быть зашифрован.
+
+
+from dz_s3.models import db, User
+from flask import Flask, render_template, request
+from dz_s3.forms import RegisterForm
+from flask_wtf.csrf import CSRFProtect
+from dz_s3.encrypt import encrypt
+
+app = Flask(__name__)
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///user.db'
+app.config['SECRET_KEY'] = b'n837tb4r8cq8473yr8q2yn398rynq0cny3r8c7ynq98c347ytn8yebuwfgr'
+csrf = CSRFProtect(app)
+
+db.init_app(app)
+
+
+@app.route('/')
+def index():
+    return 'db user!'
+
+
+@app.cli.command("init-db")
+def init_db():
+    db.create_all()
+
+
+@app.route('/register/', methods=['GET', 'POST'])
+def register():
+    form = RegisterForm()
+    if request.method == 'POST' and form.validate():
```

Comment:
Проверять запрос нет необходимости
```suggestion
    if form.validate_on_submit():
```